hackathon_site/dashboard/frontend/src/components/teamDetail/SimpleOrderTables/SimpleOrderTables.tsx

```diff
@@ -17,18 +17,25 @@ import {
     useFormikContext,
 } from "formik";
 import FormControlLabel from "@material-ui/core/FormControlLabel";
-import { OrderInTable } from "api/types";
-import { useSelector } from "react-redux";
+import { OrderInTable, OrderStatus } from "api/types";
```

Comment:
you have to do this for "Ready for Pickup" orders too (change status to Picked Up)

Reply:
There is a bug in this UI, you can't cancel an order that's ready for pickup so could you remove the "reject order" button as well